documentation/manual/src/main/paradox/algebras-and-interpreters.md

```diff
@@ -105,12 +105,12 @@ work with JSON.
 ## Interpreters
 
 Interpreters give a concrete meaning to the vocabulary and operations provided
-by the algebras. They usually rely on other libraries (e.g. circe, Akka HTTP, etc.)
+by the algebras. They usually rely on other libraries (e.g. circe, Pekko HTTP, etc.)
 to do so. Pick the interpreters that fit your existing stack!
 
 | Family                                                    | Description                                                                                                   |
 |-----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
-| @ref[Akka HTTP](interpreters/akka-http.md)                | Client and server backed by [Akka HTTP](https://doc.akka.io/docs/akka-http/current/)                          |
+| @ref[Pekko HTTP](interpreters/pekko-http.md)                | Client and server backed by [Pekko HTTP](https://pekko.apache.org/docs/pekko/current/)                          |
```

Comment:
Ultimately, we should move the Akka HTTP interpreters to their own repo and website, and here we should list both Pekko and Akka:

~~~ md
| @ref[Pekko HTTP](interpreters/pekko-http.md)         | … |
| [Akka HTTP](https://endpoints4s.github.io/akka-http) | … |
~~~